ui/app/controllers/vault/cluster/clients.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Controller from '@ember/controller';
+import { service } from '@ember/service';
+import VersionService from 'vault/services/version';
```

Comment:
nit: I'm new to typescript but I've noticed that we typically import services as `import type ...`

Reply:
oh, good catch! that's 100% right, will fix.